Tests/ServiceTierTests/LoginServiceTests.cs

```diff
@@ -32,11 +35,58 @@ public void CreateToken()
             User user = new User() { PasswordHash = "12345", Email = "test@gmail.com" };
 
             userManager.Setup(_ => _.FindByEmailAsync(model.Email)).ReturnsAsync(user);
+            userManager.Setup(_ => _.CheckPasswordAsync(user, model.Password)).ReturnsAsync(model.Password == user.PasswordHash);
             unitOfWork.Setup(_ => _.UserManager).Returns(userManager.Object);
+            jwtManager.Setup(_ => _.GenerateToken(user)).Returns(Common.Constant.Constants.JwtTokenTest);
 
             var actual = subject.CreateToken(model);
 
-            Assert.AreEqual(actual.Contains(TokenPartTest), true);
+            Assert.AreEqual(actual, Common.Constant.Constants.JwtTokenTest);
+        }
+
+        [Test]
+        public void CreateTokenWithFacebook()
+        {
+            SocialAuthDTO token = new SocialAuthDTO { AccessToken = Common.Constant.Constants.FacebookValidToken };
+            User user = new User() { PasswordHash = "12345", Email = "sashatymoshchuk07@gmail.com" };
+
+            userManager.Setup(_ => _.FindByEmailAsync(user.Email)).ReturnsAsync(user);
+            unitOfWork.Setup(_ => _.UserManager).Returns(userManager.Object);
+            jwtManager.Setup(_ => _.GenerateToken(user)).Returns(Common.Constant.Constants.JwtTokenTest);
+
+            var actual = subject.CreateTokenWithFacebook(token);
+
+            Assert.AreEqual(actual, Common.Constant.Constants.JwtTokenTest);
+        }
+
+        [Test]
+        public void CreateTokenWithFacebook_Throws_InvalidTokenException()
+        {
+            SocialAuthDTO token = new SocialAuthDTO { AccessToken = Common.Constant.Constants.FacebookInvalidToken };
+            User user = new User() { PasswordHash = "12345", Email = "sashatymoshchuk07@gmail.com" };
```

Comment:
you are using same strings multiple times - make them consts and move to the top of the file